src/assets/color-palettes.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Please change the structure. Style font is not a color. Please add values for some colors like GREEN, RED, BLUE, GREY... (according to color palette generator of @thkoehler11 )